apps/dcos_dns/src/dcos_dns.erl

```diff
@@ -170,16 +171,24 @@ push_zone(ZoneName, Records) ->
 -spec(push_prepared_zone(dns:dname(), [dns:dns_rr()]) ->
     ok | {error, Reason :: term()}).
 push_prepared_zone(ZoneName, Records) ->
+    Begin = erlang:monotonic_time(),
     try erldns_zone_cache:get_zone_with_records(ZoneName) of
         {ok, #zone{records=Records}} ->
             ok;
         _Other ->
             Hash = crypto:hash(sha, term_to_binary(Records)),
             case erldns_zone_cache:put_zone({ZoneName, Hash, Records}) of
                 ok ->
+                    Elapsed = erlang:monotonic_time() - Begin,
+                    ElapsedUs = erlang:convert_time_unit(
+                        Elapsed, native, microsecond),
                     lager:notice(
-                        "DNS Zone ~s was updated (~p records, sha: ~s)",
-                        [ZoneName, length(Records), bin_to_hex(Hash)]),
+                        "DNS Zone ~s. was updated "
```

Comment:
If you want to log `ZoneName` with `.` at the end, please do it explicitly in function arguments, not in the format string.